src/content/docs/release-notes/infrastructure-release-notes/infrastructure-agent-release-notes/new-relic-infrastructure-agent-1160.mdx

```diff
@@ -3,6 +3,8 @@ subject: Infrastructure agent
 releaseDate: '2021-03-24'
 version: 1.16.0
 ---
+⚠️ Updating RPM packages to this version will disable newrelic-infra-service. More details in [#357](https://github.com/newrelic/infrastructure-agent/issues/357)
```

Comment:
@Noly This section about Updating RPM packages is not displaying on the build. Did you want to make this a caution? If so, this is how to format that:

<Callout variant="caution">
Updating RPM packages to this version will disable newrelic-infra-service. More details in [#357](https://github.com/newrelic/infrastructure-agent/issues/357)
</Callout>

Reply:
Okey, so no caution icon right?

Reply:
Done, thanks for the feedback

Reply:
yeah. Sorry. I keep forgetting that the markdown gets rendered in these notes. I fixed it to make that clearer. (see above) And correct, no caution icon. This will set the message out in a colored box to draw attention.

Reply:
Oh, that is brilliant! Thanks! I just updated it.